tools/src/scie_pants/pants_release_tags.json

```diff
@@ -366,286 +366,5 @@
   "release_1.9.0rc0": "3afceb99a98736c3ecadb590121b3b770ed2ac02",
   "release_1.9.0rc1": "1496a9f00b9db024029a07efa90314e37a890a3f",
   "release_1.9.0rc2": "097254cb219cb8066604375d44f4889bc9700636",
-  "release_1.9.0rc3": "f8f52381e8fe982f663216a466acc0bb6e2e6bd4",
```

Comment:
The diff is deceitful, this wasn't removed. The trailing comma was (scroll WAY down)